client/packages/system/src/Item/Components/ListItems/ListItems.tsx

```diff
@@ -12,30 +13,37 @@ interface ListItemProps {
   items: ItemRowFragment[];
   route: RouteBuilder;
   enteredLineIds?: string[];
+  showNew?: boolean;
 }
 
 export const ListItems = ({
   currentItemId,
   items,
   route,
   enteredLineIds,
+  showNew = false,
 }: ListItemProps) => {
+  const t = useTranslation();
   const navigate = useNavigate();
   const value = items?.find(({ id }) => id === currentItemId) ?? null;
 
+  let options =
+    items?.map(({ id, name }) => ({
+      id,
+      value: name,
+    })) ?? [];
+  if (showNew) {
+    options.push({ id: 'new', value: t('label.new-item') });
+  }
```

Comment:
Its janky/should clean up but I tried this: https://github.com/msupply-foundation/open-msupply/compare/5750-CIV-2.5-Add-item-in-page...5750-CIV-2.5-Add-item-in-page(new-item-button)

So button is always visible, and looks different:
![image](https://github.com/user-attachments/assets/c51ed886-8a7e-43ec-9409-240a0a354727)

Reply:
I just did height 90% bc lazy but prob should be sticky haha

Reply:
Ooh nice (: Thanks